plugins/module_utils/network/sonic/config/vrrp/vrrp.py

```diff
@@ -0,0 +1,719 @@
+                    if not match_group:
+                        add_cfg.setdefault('group', []).append(group)
+                    else:
+                        add_group, del_group = {}, {}
+                        for attr in self.vrrp_attributes:
+                            if attr in group:
+                                if attr not in match_group:
+                                    add_group[attr] = group[attr]
+                                elif group[attr] != match_group[attr]:
+                                    if attr == 'virtual_address':
+                                        add_vip, del_vip = [], []
+                                        want_vip = set(self.get_vip_addresses(group[attr]))
+                                        match_vip = set(self.get_vip_addresses(match_group[attr]))
+                                        for vip in want_vip.difference(match_vip):
+                                            add_vip.append({'address': vip})
+                                        for vip in match_vip.difference(want_vip):
+                                            del_vip.append({'address': vip})
+                                        if add_vip:
+                                            add_group[attr] = add_vip
+                                        if del_vip:
+                                            del_group[attr] = del_vip
+                                    elif attr == 'track_interface':
+                                        add_track, del_track = [], []
+                                        for track in group[attr]:
+                                            match_track = next((t for t in match_group[attr] if t['interface'] == track['interface']), None)
+                                            if not match_track:
+                                                add_track.append(track)
+                                            elif track['priority_increment'] != match_track['priority_increment']:
+                                                add_track.append(track)
+                                                del_track.append(match_track)
+                                        for match_track in match_group[attr]:
+                                            track = next((t for t in group[attr] if t['interface'] == match_track['interface']), None)
+                                            if not track:
+                                                del_track.append(match_track)
+                                        if add_track:
+                                            add_group[attr] = add_track
+                                        if del_track:
+                                            del_group[attr] = del_track
+                                    else:
+                                        add_group[attr] = group[attr]
+                                        del_group[attr] = match_group[attr]
```

Comment:
Since the option is being updated, the delete can be removed.

Reply:
Removed delete